angular/projects/spark-angular/src/public-api.ts

```diff
@@ -240,3 +252,11 @@ export { SprkTooltipComponent } from './lib/components/sprk-tooltip/sprk-tooltip
 export { SprkTooltipModule } from './lib/components/sprk-tooltip/sprk-tooltip.module';
 export { SprkDividerDirectiveModule } from './lib/directives/sprk-divider/sprk-divider.module';
 export { SprkDividerDirective } from './lib/directives/sprk-divider/sprk-divider.directive';
+export { SprkTextareaDirective } from './lib/directives/inputs/sprk-textarea/sprk-textarea.directive';
+export { SprkTextareaModule } from './lib/directives/inputs/sprk-textarea/sprk-textarea.module';
+export { SprkTabsComponent } from './lib/components/sprk-tabs/sprk-tabs.component';
+export { SprkTabsModule } from './lib/components/sprk-tabs/sprk-tabs.module';
+export { SprkTabsPanelDirective } from './lib/directives/sprk-tabs/sprk-tabs-panel/sprk-tabs-panel.directive';
+export { SprkTabsPanelModule } from './lib/directives/sprk-tabs/sprk-tabs-panel/sprk-tabs-panel.module';
```

Comment:
Verify if every new component we have added that supposed to be is, is. @guern1kn

Reply:
Confirmed.